testsuite/MDAnalysisTests/analysis/test_dihedrals.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from numpy.testing import assert_array_almost_equal
```

Comment:
please use `assert_almost_equal`. It does some more equality checks then `assert_array_almost_equal`